app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java

```diff
@@ -243,13 +243,13 @@ public void onDismissed(Snackbar snackbar, int event) {
                                 fdelete.delete();
                                 mDatabaseHelper.insertRecord(fdelete.getAbsolutePath(),
                                         mActivity.getString(R.string.deleted));
+                                if (mFileList.size() == 0) {
+                                    mEmptyStateChangeListener.setEmptyStateVisible();
+                                }
                             }
                         }
                     }).show();
         }
-        if (mFileList.size() == 0) {
```

Comment:
Shouldn't this piece of code be here also?

Suppose, there is only one file & I delete it. The empty view won't be shown at first.



You can do this also : Add one more function `setEmptyStateInvisible` in the corresponding interface and call that function when file delete undo is clicked.

Reply:
Ok